src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/w3c/JavascriptW3cPerformanceTimeline.kt

```diff
@@ -1,117 +1,31 @@
 package com.atlassian.performance.tools.jiraactions.api.w3c
 
+import com.atlassian.performance.tools.jiraactions.api.w3c.harvesters.getJsElementsPerformance
+import com.atlassian.performance.tools.jiraactions.api.w3c.harvesters.getJsNavigationsPerformance
+import com.atlassian.performance.tools.jiraactions.api.w3c.harvesters.getJsResourcesPerformance
 import org.openqa.selenium.JavascriptExecutor
-import java.time.Duration
 
 /**
  * Obtains entries from [javascript].
  */
 class JavascriptW3cPerformanceTimeline(
-    private val javascript: JavascriptExecutor
+    javascript: JavascriptExecutor,
+    withNavigationPerformance: Boolean = true,
```

Comment:
I was thinking about decorators, but a good old `Boolean` seems to be a way better fit. Nice one!